event/receiver.go

```diff
@@ -0,0 +1,30 @@
+package event
+
+import (
+	"github.com/ONSdigital/go-ns/kafka"
+	"github.com/ONSdigital/dp-import-reporter/model"
+	"github.com/ONSdigital/dp-import-reporter/schema"
+	"github.com/ONSdigital/go-ns/log"
+)
+
+type EventHandler interface {
+	HandleEvent(e *model.ReportEvent) error
+}
+
+type Receiver struct {
+	Handler EventHandler
+}
+
+func (r *Receiver) ProcessMessage(msg kafka.Message) error {
+	var reportEvent model.ReportEvent
```

Comment:
There is discussion about making this service handle both import & export errors. If we do decide to go in that direction then there would be a simple witch in here to determined which "service" to call based on the message type